boards/arm/nrf9160dk_nrf9160/doc/index.rst

```diff
@@ -156,6 +156,11 @@ image. The Secure image can be built using either Zephyr or
 `Trusted Firmware M`_ (TF-M). Non-Secure firmware images are always built
 using Zephyr. The two alternatives are described below.
 
+.. note::
+
+   By default the the Secure image for nRF5340 application core is built
```

Comment:
5340 -> 9160

Reply:
done

Reply:
still nRF5340

Reply:
Should be fixed now, thanks @jfischer-no